### DISCUSSION TITLE
How to close the custom action button?

### DISCUSSION THREAD

**Discussion by tallano** [geoman-io/leaflet-geoman#1129] [Questions & Help] [answered]
Hi,
I'm using custom actions on a custom control I created, but I can't find how to close the custom action button when clicked and having the control button go back to its unclicked state.
Any ideas? :)

**Comment by tallano** [comment, marked as answer]
Ok, I found a solution:

`map.pm.Toolbar.buttons['myButtonName']._triggerClick();`

Don't' know if it is the cleanest or the recommended way, but it works.

**Comment by ludik0** [comment]
Thank you for the solution. 

`map.pm.Toolbar.buttons['myButtonName'].toggle();`

Does also work.
